spec/system/groups/group_spec.rb

```diff
@@ -63,6 +63,21 @@
     end
   end
 
+  describe "update a group" do
```

Comment:
```suggestion
  describe "create a group" do
```

?

Reply:
There is already a test for creating. This problem only affected updating. 🙂

Reply:
But the description is "it creates a new group"... am I missing something? :sweat_smile: